components/DemoAnimation.js

```diff
@@ -216,10 +217,13 @@ export default function DemoAnimation({ className = '' }) {
         </div>
         <div className={`relative md:flex-1 md:ml-6 md:mb-0 transition-all duration-500 ease-in-out z-10`}>
           <div className={`md:text-left text-center mt-8 md:mt-0 lg:absolute lg:left-0 lg:top-0 lg:right-0 lg:ml-48 lg:mr-8 ${showControls ? 'block' : 'hidden'}`}>
-            <h3 className="text-primary-800 text-2xl font-bold md:text-2xl mb-4">
-              Play with it!
-            </h3>
-            <p className="text-gray-500 text-lg font-normal mb-6 max-w-3xl mx-auto mb-8">
+            <Heading
+            style="h3"
+            text="Play with it!"
+            level="h3"
+            className="mb-4"
+            />
```

Comment:
I know that we don't have a integrated `eslint` but please make indentation for horizontal props:

```
            <Heading
              style="h3"
              text="Play with it!"
              level="h3"
              className="mb-4"
            />
```

React components are treated as objects and indentation in objects is desirable in JS world :)

Please fix it here and in other places.

Reply:
Got it! Will update that across all files and noted for the future, thanks!